src/reactapp/src/components/FetchHelpers.js

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+import { transform as _transform } from "lodash-es";
+
+// TODO: replace hard-coded with some sort of runtime setting
+const BaseUrl = "https://localhost:7065";
```

Comment:
I forgot to mention the ticket number in the `TODO` comment. Whenever you have known technical debt - a violation of the team's standards - then it is best to create a ticket right away and mention it in the code.
```suggestion
// Will be fixed in EPPETA-19.
const BaseUrl = "https://localhost:7065";
```